packages/logger/tests/unit/helpers.test.ts

```diff
@@ -56,7 +56,7 @@ describe('Helper: createLogger function', () => {
     test('when no parameters are set, returns a Logger instance with the correct properties', () => {
 
       // Prepare
-      const loggerOptions = {
+      const loggerOptions: ConstructorOptions = {
```

Comment:
We probably need also tests for lower case configuration of log level.

Reply:
Thank you for pointing this out, by adding the test I found out that I was introducing a bug.

[This commit](https://github.com/awslabs/aws-lambda-powertools-typescript/pull/1313/commits/5488c0af993d299fa3624be459077ad07155e1ca) adds the test and fixes the logic.